DesafioDesignPatterns-N1/DAO/Bridge.swift

```diff
@@ -0,0 +1,73 @@
+//
+//  Bridge.swift
+//  DesafioDesignPatterns-N1
+//
+//  Created by Jonattan Moises Sousa on 20/05/21.
+//
+
+import UIKit
+
+class Bridge {
```

Comment:
Seguem dois exemplos do uso deste Design Patterns
https://github.com/ochococo/Design-Patterns-In-Swift/blob/master/source/structural/bridge.swift
https://medium.com/swiftworld/swift-world-design-patterns-bridge-a20bbe999059

Observe como ele é chamado em ambos exemplos.